### PR TITLE
Patch 89

### PR DESCRIPTION
**Pull request by elnur004**

### CODE REVIEW

**Comment by vadim9999** [comment]
@elnur004 can you wrap in array all object in json ?

**Comment by elnur004** [comment]
@vadim9999 hi.
there was a problem with the internet. i'm just now can be online.
all json files should be in an array or just objects in the cooking-times.json file should be in an array?

**Comment by vadim9999** [comment]
@elnur004 all objects in cooking-times.json and all jsons files that you added

File: src/data/Cooking_times/cooking_times.json [hunk 1]
```diff
@@ -5,104 +5,131 @@
             "microwave": "4 - 6 mins",
             "steam": "8 - 12 mins",
             "roast": "25 - 30 mins"
-           },
-
+           }
+         ],                      
+       
+         [
```
Comment (vadim9999) [inline src/data/Cooking_times/cooking_times.json]: No, I mean one array and many objects, please change it

**Comment by elnur004** [comment]
is that way correct?

**Pull request merged**